src/content/advanced/iam-roles-for-service-accounts/index.md

```diff
@@ -328,3 +328,49 @@ If you have followed the steps above you can switch from kiam to IAM roles for s
 - Verify that your application is working correctly.
 
 In case your application is not working you can always remove the annotation on the service account and add the kiam annotation on your deployment again.
+
+## AWS Release v19
+
+When upgrading to AWS Release `v19.0.0` or higher, we have made changes to the identity provider URL to provide greater predictability and enable easier automation of AWS IAM role creation by customers. Specifically, we have decided to use a `Cloudfront domain alias` for the identity provider URL.
+
+Previously, in AWS Release `v18.x.x.`, the default Cloudfront domain name - a randomly generated URL such as `https://d111111abcdef8.cloudfront.net` - was used as the identity provider URL. However, with the introduction of the Cloudfront alternate domain name, the new identity provider URL will be `https://irsa.CLUSTER_ID.k8s.INSTALLATION_NAME.BASE_DOMAIN`.
+
+**Please note that before upgrading to release `v19.0.0` or higher, it is necessary to update your AWS IAM roles with the new identity provider URL.**
+
+To find the new URL, simply log into the AWS Console, navigate to IAM > Identity Providers, and search for the updated identity provider URL there.
```

Comment:
can they check it on MC as well?

Reply:
No, domain alias will only be written into the IRSA config map once they upgrade to v19.

Reply:
approving then